gulp/tasks/sass.js

```diff
@@ -15,19 +16,23 @@ module.exports = function() {
 
     CustomBuildUtil.checkOptions(argv);
 
-    fs.writeFileSync('styles/selectivity-custom.less', argv.modules.map(function(module) {
-        if (fs.existsSync('styles/selectivity-' + module + '.less')) {
-            return '@import "selectivity-' + module + '";';
-        } else {
-            return '';
-        }
+    // Config files!
+    argv.sassModules = argv.modules.slice();
```

Comment:
No need to assign this to the argv object. `var sassModules = ...` would do the trick, I think.